### ISSUE TITLE
Text view / Multiline text field

### ISSUE THREAD

**Issue by MaxHasADHD** [mamaral/MAFormViewController#9]
A multiline text field or a text view with a set/resizable size would be good

**Comment by pashikua** [comment]
+1

**Comment by mamaral** [comment]
I'll try and get this out tonight.

**Comment by mamaral** [comment]
I now recall why I didn't do this in the first place - text views don't have placeholders, so I'll have to implement some alternative.

**Comment by pashikua** [comment]
@mamaral You can use SZTextView

**Comment by mamaral** [comment]
@pashikua Yeah, I'm aware of a few alternatives, although I'd prefer to not have any dependencies as I'm animating placeholders and stuff, so I'd rather have some more control over it. I'd like to roll my own - we'll see.

**Comment by dippol** [comment]
So, how to use textview in the MAFormViewController? Is enhancement done?

**Comment by mamaral** [comment]
No, this hasn't been added and honestly likely won't be. I'm not sure how much more support will be kept on this library in the future. It's still TBD.

**Comment by dippol** [comment]
What a pity!
Control is a great!

Dmitry
